bin/ncp/TOOLS/db-optimize.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Optimize MariaDB
+#
+
+configure()
+{
+  echo -ne "begin of MariaDB Optimization..."
+  mariadb-check --optimize --all-databases >/dev/null
```

Comment:
This command doesn't work for me without setting the user and also I think it's good to show users the output. Can you change it to:

```diff
-   mariadb-check --optimize --all-databases >/dev/null
+   mariadb-check -u root --optimize --all-databases
```

Reply:
Makes sense. I've changed file accordingly.

Reply:
Thank you, looks good :)